tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml

```diff
@@ -51,7 +51,6 @@
      <standardCssUrl>tds.css</standardCssUrl>

```

Comment:
Should these be changed to match what you have in the docs?

~~~xml
<htmlSetup>
   <standardCssUrl>standard.css</standardCssUrl>
   <catalogCssUrl>catalog.css</catalogCssUrl>
   <datasetCssUrl>dataset.css</datasetCssUrl>
</htmlSetup>
~~~